apps/omg/test/omg/dependency_conformance/signature_test.exs

```diff
@@ -43,24 +40,46 @@ defmodule OMG.DependencyConformance.SignatureTest do
     {:ok, [addr | _]} = Ethereumex.HttpClient.eth_accounts()
 
     {:ok, _, signtest_addr} =
-      Eth.Deployer.create_new(OMG.Eth.Eip712SignatureWrapper, root_path, Eth.Encoding.from_hex(addr), [])
+      Eth.Deployer.create_new("PaymentEip712LibMock", root_path, Eth.Encoding.from_hex(addr), [])
+
+    # impose our testing signature contract wrapper (mock) as the validating contract, which normally would be
+    # plasma framework
+    :ok = Application.put_env(:omg_eth, :contract_addr, %{plasma_framework: Eth.Encoding.to_hex(signtest_addr)})
 
-    # TODO if this breaks here someday, it might be because this entry has been changed to be a map of multiple addrs
-    :ok = Application.put_env(:omg_eth, :contract_addr, Eth.Encoding.to_hex(signtest_addr))
+    on_exit(fn ->
+      # reverting to the original values from `omg_eth/config/test.exs`
+      Application.put_env(:omg_eth, :contract_addr, %{plasma_framework: "0x0000000000000000000000000000000000000001"})
+      exit_fn.()
+    end)
 
-    on_exit(exit_fn)
     [contract: signtest_addr]
   end
 
   test "signature test empty transaction", context do
     contract = context[:contract]
-    tx = Transaction.Payment.new([], []) |> DevCrypto.sign([@alice.priv])
-    sig = tx.sigs |> Enum.at(0)
+    tx = TestHelper.create_signed([], [])
+    verify(contract, tx)
+  end
 
-    verify(contract, tx, sig)
+  test "no inputs test", context do
+    contract = context[:contract]
+    tx = TestHelper.create_signed([], [{@alice, @eth, 100}])
+    verify(contract, tx)
   end
 
-  test "signature test", context do
+  test "no outputs test", context do
+    contract = context[:contract]
+    tx = TestHelper.create_signed([{1, 0, 0, @alice}], [])
+    verify(contract, tx)
+  end
+
+  test "signature test - small tx", context do
+    contract = context[:contract]
+    tx = TestHelper.create_signed([{1, 0, 0, @alice}], [{@alice, @eth, 100}])
```

Comment:
is there a way around using TestHelpers? I think they're somehow becoming a bucket for everything and make tests that should be self sufficient into a tree of jumps between modules and functions.

Reply:
perhaps something for a separate issue? Maybe let's propose a bunch of renames and split that particular module into submodules (entities, txs and fee-stuff separately), if it's really bugging you.

Note this PR doesn't really change the approach it just makes it consistent (use `create_signed` in all cases)